kitchen.yml

```diff
@@ -10,15 +10,14 @@ provisioner:
   name: chef_zero
   chef_license: accept
   product_name: chef
-  require_chef_omnibus: 15
+  produc_version: 15
```

Comment:
Any reason this needs to be pinned at 15?
```suggestion
  product_version: 15
```

Reply:
Changed to latest.

Reply:
Updated to latest.